### PR TITLE
es-blog-heading: Base component

### PR DESCRIPTION
**Pull request by pbishop16**
Addresses #19 

Usage:
---
- Blog heading without a tag
image
```
{{es-blog-heading
  author="Sara Poster"
  postDate="03-20-2018"
  postUrl="https://emberjs.com/blog"
  postTitle="Pretty Awesome Blog Post"
}}
```
- Blog heading with a guest tag
image
```
{{es-blog-heading
  author="John Blogger"
  postType="guest"
  postDate="03-20-2018"
  postUrl="https://emberjs.com/blog"
  postTitle="Pretty Awesome Blog Post"
}}
```
- Blog heading with a syndicated tag
image
```
{{es-blog-heading
  author="Samantha Blogger"
  postType="syndicated"
  postDate="03-20-2018"
  postUrl="https://emberjs.com/blog"
  postTitle="Pretty Awesome Blog Post"
}}
```
- Blog heading on the individual blog's page (i.e. without url provided)
image
```
{{es-blog-heading
  author="Brian Poster"
  postType="syndicated"
  postDate="03-20-2018"
  postTitle="Pretty Awesome Blog Post"
}}
```

image

### CODE REVIEW

**Comment by pbishop16** [comment]
@MelSumner Please let me know if you would like the class names to match those used on the current site.

File: addon/styles/_es-blog-heading.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+h1.post-title {
+  color: $secondary;
+  font-size: 2.25em;
+  margin-bottom: 0.01em;
+	margin-top: 0.7em;
+}
+
+h2.post-title {
```
Comment (MelSumner) [inline addon/styles/_es-blog-heading.scss]: What happens if you don't have the h1 or h2 here, and just have the classes?

File: addon/styles/_es-blog-heading.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
+h1.post-title {
```
Comment (MelSumner) [inline addon/styles/_es-blog-heading.scss]: So the component name should be added to the component as a class name. 
that means here in your scss, you can do this: 
.es-blog-heading {
  // scoped styles here
}

File: addon/components/es-blog-heading.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import {
+  BlogPostType,
+} from '../constants/es-blog-heading';
+import {
+  computed,
+} from '@ember/object';
+
+export default Component.extend({
+  layout,
+  classNames: ['post-header'],
```
Comment (MelSumner) [inline addon/components/es-blog-heading.js]: the component name should be added here as the class name: 'es-blog-heading'

File: addon/styles/_es-blog-heading.scss (hunk 1 above)
Comment (pbishop16) [reply addon/styles/_es-blog-heading.scss]: @MelSumner Yeah, I can definitely do that.

File: addon/styles/_es-blog-heading.scss (hunk 2 above)
Comment (pbishop16) [reply addon/styles/_es-blog-heading.scss]: @MelSumner Makes sense. Will do.

File: addon/components/es-blog-heading.js (hunk 3 above)
Comment (pbishop16) [reply addon/components/es-blog-heading.js]: @MelSumner Absolutely will do.

**Pull request merged**